project.clj

```diff
@@ -8,6 +8,7 @@
                  [clj-time              "0.11.0"]
                  [org.clojure/data.json "0.2.6"]
                  [org.clojure/data.csv  "0.1.3"]
+                 [clj-aws-s3            "0.3.10"]
```

Comment:
I'd recommend using amazonica rather than clj-aws-s3 'cos it uses the newer aws sdk under the covers which works in all regions.